src/resolvers.ts

```diff
@@ -2,41 +2,71 @@ import * as facilityService from './services/facilityService'
 import * as healthcareProfessionalService from './services/healthcareProfessionalService'
 import * as gqlType from './typeDefs/gqlTypes'
 import * as submissionService from './services/submissionService'
+import CustomErrors from './errors'
 
 const resolvers = {
     Query: {
         facilities: async () => {
-            const matchingFacilities = await facilityService.searchFacilities(['1'])
+            try {
+                const matchingFacilities = await facilityService.searchFacilities(['1'])
 
-            return matchingFacilities
+                return matchingFacilities
+            } catch (error) {
+                return CustomErrors.notFound(`Failed to find facilities: ${error}`)
```

Comment:
We don't actually want to return errors back to the user as it's a security risk and not helpful to them. 
We should 
1) log the error and user request for our own debugging (console.log is fine for now. I'll be moving all of them to real logs later), 
then 2) return a helpful message to the user. 

Let's update each of the try/catches with this. 
Consider what message you want to send and what action the user should take.

Reply:
By user request do you mean the arguments or the request body (contextValue) for example:
```
{
  "healthcareProfessionalId": "HhnkvZnpY3xWA7snihhr"
}
```